lsp-mode.el

```diff
@@ -5388,41 +5388,41 @@ returns the command to execute."
 (defun lsp--create-sentinel (workspace)
   "Create sentinel handler for WORKSPACE."
   (lambda (process exit-str)
-    (let* ((status (process-status process))
-           (folder->workspaces (lsp-session-folder->servers (lsp-session)))
-           (stderr (-> workspace lsp--workspace-proc process-name get-buffer)))
+    (unless (process-live-p process)
```

Comment:
Is there any early-return mechanism to reduce indentation below?

Reply:
AFAIK there is early return using cl-lib, I refactored the code to use closure.